src/components/WelcomeText.js

```diff
@@ -16,12 +19,12 @@ const defaultProps = {
 const WelcomeText = props => (
     <>
         <Text style={[props.textSize === 'large' ? styles.textP : styles.textLabel, styles.textStrong, styles.mb1]}>
-            With Expensify.cash, chat and payments are the same thing.
+            {props.translations.translate('welcomeText')[0]}
```

Comment:
This works? I thought it did not...

Reply:
It works. Would you suggest a different approach?

Reply:
Not sure if we should allow it, what happens if one language does not define all the array indexes you expect? It would silently fail or show nothing. If we used somethig like `props.translations.translate('welcomeText.phrase1')` then the library would apply the proper fallbacks (like getting the `en` translation if we are in `en-UK` and there are no translations for that specific locale) and alert us when necessary